src/assets/svg/4G.svg

```diff
@@ -1,6 +1,6 @@
-<svg width="41" height="40" viewBox="0 0 41 40" fill="none" xmlns="http://www.w3.org/2000/svg">
-  <path d="M20.7841 30.234C22.4941 30.234 24.5101 29.82 26.2561 28.218L27.1381 22.854H22.3321L21.9361 25.23H23.7541L23.4841 26.904C22.8901 27.282 22.2601 27.426 21.5221 27.426C19.8481 27.426 19.1101 26.31 19.1101 24.546C19.1101 21.702 20.3881 19.686 22.5841 19.686C23.8981 19.686 24.2401 20.424 24.1861 21.342H27.3901C27.6241 18.768 26.5801 16.824 22.9981 16.824C18.1021 16.824 15.7981 20.604 15.7981 25.05C15.7981 27.858 17.0761 30.234 20.7841 30.234Z" fill="currentColor"/>
-  <path d="M7.56407 24.762L10.6781 20.82L10.0301 24.762H7.56407ZM9.14807 30H12.7121L13.1081 27.606H14.2241L14.7101 24.762H13.5941L14.7821 17.58H10.3001L4.48607 24.834L4.01807 27.606H9.54407L9.14807 30Z" fill="currentColor"/>
-  <path d="M20.9712 12.9156C23.1027 12.0368 25.5618 12.1019 27.6673 13.3175C29.7721 14.5327 31.0588 16.6303 31.3635 18.9156" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-  <path d="M19.0001 8.33013C22.5526 6.86555 26.6511 6.9739 30.1604 9C33.6683 11.0253 35.8128 14.5213 36.3206 18.3301" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
still has width and height attributes in this file

Reply:
i updated it ;-)